freedomofpress/securedrop-client
# move req files to new dir

# Description

Move requirement file into a requirements directory

# Test Plan

0. delete or `.venv`
1. run `make venv`

    - [ ] verify that you see: [venv.txt](https://github.com/freedomofpress/securedrop-client/files/4956509/venv.txt)

2. activate the newly created environment: `source .venv/bin/activate`
3. run the client

    - [ ] confirm that you see login window

4. run `make safety`

    - [ ] verify that you see: [safety.txt](https://github.com/freedomofpress/securedrop-client/files/4956505/safety.txt)

5. run `make update-pip-requirements`

    - [ ] verify that you see: [update-pip-requirements.txt](https://github.com/freedomofpress/securedrop-client/files/4956503/update-pip-requirements.txt)

6. run `make update-mac-pip-requirements` 

    - [ ] verify that you see the same results as you do on `main` branch (i will confirm if you don't have a mac - just let me know)

7. build a debian package from the `update-client-reqs-location` packaging branch that supports a requirements directory (see https://github.com/freedomofpress/securedrop-debian-packaging/pull/183): 

    - [ ] package builds without error
    - [ ] run that package and confirm that you see login window

# Checklist

 - [ ] Once https://github.com/freedomofpress/securedrop-debian-packaging/pull/183 is merged verify that ci passes here
 - No update to the AppArmor profile is required for these changes
 - No database schema changes are needed
